build/build.script.mk

```diff
@@ -1,7 +1,13 @@
-# Common makefile for installing userspace scripts.
+# vim: tabstop=8 shiftwidth=8 noexpandtab:
 #
-# This template assumes following make variables are set:
-#  SCRIPT: The name of script.
+# This is a common makefile for installing userspace scripts.
+#
+# This following make variables are assumed to be set:
+# -SCRIPT: The name of script.
+# -BINDIR: The path realtive to $(SYSROOT) at which $(SCRIPT) will be 
+#  installed. Defaults to $(SYSROOT)/bin.
```

Comment:
If it's already relative to `$(SYSROOT)`, then shouldn't the default value be just `bin`?